gpfit/fit.py

```diff
@@ -77,36 +77,48 @@ def fit(xdata, ydata, K, ftype="ISMA"):
     A = params[[i for i in range(K*(d+1)) if i % (d + 1) != 0]]
     B = params[[i for i in range(K*(d+1)) if i % (d + 1) == 0]]
 
+    cs = []
+    exps = []
     if ftype == "ISMA":
         alpha = 1./params[range(-K, 0)]
+        for k in range(K):
+            cs.append(exp(alpha[k]*B[K]))
```

Comment:
@bqpd I reverted back to this because I was having problems with higher dimensions using lhs, and rhs.